src/sql/knexfile.js

```diff
@@ -18,16 +18,13 @@ const recursiveCamelcase = (result) => _.transform(result, (acc, value, key, tar
 });
 
 // This is one of the shapes the knexfile can take https://knexjs.org/#knexfile
-const fetchConfiguration = async (environment, rdsSandboxId) => {
-  const params = await getConnectionParams(environment, rdsSandboxId);
-  return {
-    [environment]: {
-      client: 'postgresql',
-      connection: params,
-      postProcessResponse: recursiveCamelcase,
-    },
-  };
-};
+const fetchConfiguration = async (environment, rdsSandboxId) => ({
+  [environment]: {
+    client: 'postgresql',
+    connection: async () => await getConnectionParams(environment, rdsSandboxId),
+    postProcessResponse: recursiveCamelcase,
+  },
+});
```

Comment:
This is the important change: don't call `await` of `getConnectionParams`, pass an async function that knex can `await` when necessary instead